test/functional/test.sh

```diff
@@ -44,7 +44,7 @@ do
     cp -r $site_convert/non_markbind_site/_site $site_convert
 
     # run our test script to compare html files
-    node testUtil/test.js $site_convert
+    node testUtil/test.js compare $site_convert
```

Comment:
`test/functional/testUtil/test.js` doesn't seem to be using `compare` as the command keyword to run the tests, so how does this work? 🤔

Reply:
I extended the functionality `test.js` to take in actions and flags like file path.

Reply:
I understand your purpose of extending the `test.js`. What I am saying is that your extended `test.js` is using the keyword `test`, not `compare`:

https://github.com/MarkBind/markbind/blob/7f07b5ce9db10d3912783ccd4300b1711ad09d9f/test/functional/testUtil/test.js#L14-L15

So nothing in `test.js` is actually getting executed. Calling a non-existent command also returns 0, so we don't know whether the tests work or not. There is also another issue with `test.js`, which I highlighted in another comment...